server/server.go

```diff
@@ -293,6 +304,19 @@ func (w *otherWorker) Do(job interface{}) interface{} {
 		w.s.stats.logRequestDuration(requestBegin)
 		return makeRespondResponse(pkt.ID, res)
 
+	case protocol.OpRPC:
+		codec := newServerCodec(pkt.Payload)
+
+		err := w.s.dispatcher.ServeRequest(codec)
+		if err != nil {
+			log.Errorf("Worker %v: ServeRPC: %v", w.name, err)
+			w.s.stats.logInvalid(requestBegin)
+			return makeErrResponse(pkt.ID, protocol.ErrInternal)
+		}
+
+		w.s.stats.logRequestDuration(requestBegin)
+		return makeRespondResponse(pkt.ID, codec.response)
+
```

Comment:
Sorry, my point is that all non-ECDSA work is currently done in a single worker pool, so a sufficient number of these requests, if they blocked for a meaningful amount of time, could block all other non-ECDSA operations on the server. So my question is: are we expecting these RPC servers to be run on the same machine as the Keyless server, or might they be run elsewhere across the network?